### PR TITLE
added pg:links

### PR DESCRIPTION
**Pull request by jdx**

### CODE REVIEW

File: commands/links/destroy.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    yield heroku.delete(`/client/v11/databases/${db.name}/links/${encodeURIComponent(args.link)}`, {host: host(db)})
+  }))
+}
+
+module.exports = {
+  topic: 'pg',
+  command: 'links:destroy',
+  description: 'destroys a link between data stores',
+  help: `Example:
+  heroku pg:links:create HEROKU_POSTGRESQL_CERULEAN redis_symmetrical_100`,
```
Comment (msakrejda) [inline commands/links/destroy.js]: this should be `pg:links:destroy`, no? Also the link target name is not a standard format we use--should this be something like `redis-symmetrical-100` instead?

File: test/commands/links/create.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    nock.cleanAll()
+    api.done()
+    pg.done()
+  })
+
+  it('creates a link', () => {
+    pg.post('/client/v11/databases/postgres-1/links', {target: 'postgres-1', as: 'foobar'}).reply(200, {name: 'foobar'})
+    return cmd.run({app: 'myapp', args: {}, flags: {confirm: 'myapp', as: 'foobar'}})
+    .then(() => expect(cli.stdout, 'to equal', ''))
+    .then(() => expect(cli.stderr, 'to equal', 'Adding link from postgres-1 to postgres-1... foobar\n'))
```
Comment (msakrejda) [inline test/commands/links/create.js]: This reads kinda weird... should we  change this to `... done; added foobar` or `... added foobar` or something? /cc @neovintage

**Review by msakrejda — CHANGES_REQUESTED** [review]

File: commands/links/destroy.js (hunk 1 above)
Comment (jdx) [reply commands/links/destroy.js]: yes to both, fixed

File: test/commands/links/create.js (hunk 2 above)
Comment (jdx) [reply test/commands/links/create.js]: 👍  to making it more clear, updated

**Review by msakrejda — APPROVED** [review]

**Pull request merged**